packages/grid-pro/src/vaadin-grid-pro-edit-column.js

```diff
@@ -286,6 +290,23 @@ class GridProEditColumn extends GridColumn {
     this._focusEditor(editor);
   }
 
+  _onMouseDown(editor) {
+    return (e) => {
+      this.__mouseDownDebouncer = Debouncer.debounce(this.__mouseDownDebouncer, animationFrame, () => {
+        if (e.target !== editor.inputElement) {
+          this._focusEditor(editor);
+          this._grid._cancelStopEdit();
+        }
+      });
+    };
+  }
+
+  _onMouseUp() {
+    if (this.__mouseDownDebouncer) {
+      this.__mouseDownDebouncer.cancel();
+    }
+  }
```

Comment:
I added this call to cancel debouncer, but in my tests it never gets called. Maybe can be removed.